<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>List_Design</class>
 <widget class="QWidget" name="List_Design">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="kill_Button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>KILL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="view_Button">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>XEM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_Button">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>XÃ“A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start_Button">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>START</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="list_app">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>341</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Name Application</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ID Application</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Count Threads</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
